### PR TITLE
Menu tasks

### PR DESCRIPTION
**Pull request by dkorhov**

### CODE REVIEW

File: docroot/sites/all/modules/custom/menu_tasks/menu_tasks.module [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+
+/**
+ * Callback for pages 'training/menu/title'.
+ *
+ * @return string
+ *   returns custom title based on username.
+ */
+function menu_tasks_training_title_callback() {
+  global $user;
+  $username = $user->name ? $user->name : 'Anonymous';
```
Comment (dprudnikov) [inline docroot/sites/all/modules/custom/menu_tasks/menu_tasks.module]: Seems to me for anonymous user object $user->name will generate notice

File: docroot/sites/all/modules/custom/menu_tasks/menu_tasks.module [hunk 2]
```diff
@@ -0,0 +1,319 @@
+<?php
+
+/**
+ * @file
+ * Module with menu tasks.
+ */
+
+/**
```
Comment (dprudnikov) [inline docroot/sites/all/modules/custom/menu_tasks/menu_tasks.module]: Place hook_permission below hook_menu

File: docroot/sites/all/modules/custom/menu_tasks/menu_tasks.module (hunk 1 above)
Comment (dkorhov) [inline docroot/sites/all/modules/custom/menu_tasks/menu_tasks.module]: You were right. Fixed in 6c43e11

File: docroot/sites/all/modules/custom/menu_tasks/menu_tasks.module (hunk 2 above)
Comment (dkorhov) [inline docroot/sites/all/modules/custom/menu_tasks/menu_tasks.module]: Changed in 6c43e11

**Pull request merged**